pyaedt/modules/PostProcessor.py

```diff
@@ -4911,3 +4915,455 @@ def sample_ami_waveform(
+        self._app.osolution.EditFieldsSummarySetting(arg)
+
+
+class IcepakPostProcessor(PostProcessor, object):
+    def __init__(self, app):
+        PostProcessorCommon.__init__(self, app)
+
+    @pyaedt_function_handler()
+    def create_field_summary(self):
+        return FieldSummary(self._app)
+
+    @pyaedt_function_handler()
+    def get_fans_operating_point(self, export_file=None, setup_name=None, timestep=None, design_variation=None):
+        """
+        Get the operating point of the fans in the design.
+
+        Parameters
+        ----------
+        export_file : str, optional
+            Name of the file to save the operating point of the fans. The default is
+            ``None``, in which case the filename is automatically generated.
+        setup_name : str, optional
+            Setup name to determine the operating point of the fans. The default is
+            ``None``, in which case the first available setup is used.
+        timestep : str, optional
+            Time, with units, at which to determine the operating point of the fans. The default
+            is ``None``, in which case the first available timestep is used. This parameter is
+            only relevant in transient simulations.
+        design_variation : str, optional
+            Design variation to determine the operating point of the fans from. The default is
+            ``None``, in which case the nominal variation is used.
+
+        Returns
+        -------
+        list
+            First element of the list is the CSV filename. The second and third elements
+            are the quantities with units describing the operating point of the fans.
+            The fourth element is a dictionary with the names of the fan instances
+            as keys and lists with volumetric flow rates and pressure rise floats associated
+            with the operating points as values.
+
+        References
+        ----------
+
+        >>> oModule.ExportFanOperatingPoint
+
+        Examples
+        --------
+        >>> from pyaedt import Icepak
+        >>> ipk = Icepak()
+        >>> ipk.create_fan()
+        >>> filename, vol_flow_name, p_rise_name, op_dict= ipk.get_fans_operating_point()
+        """
+
+        if export_file is None:
+            path = self._app.temp_directory
+            base_name = "{}_{}_FanOpPoint".format(self._app.project_name, self._app.design_name)
+            export_file = os.path.join(path, base_name + ".csv")
+            while os.path.exists(export_file):
+                file_name = generate_unique_name(base_name)
+                export_file = os.path.join(path, file_name + ".csv")
+        if setup_name is None:
+            setup_name = "{} : {}".format(self._app.get_setups()[0], self._app.solution_type)
+        if timestep is None:
+            timestep = ""
+            if self._app.solution_type == "Transient":
+                self._app.logger.warning("No timestep specified. First timestep will be exported.")
```

Comment:
```suggestion
                self._app.logger.warning("No timestep is specified. First timestep is exported.")
```
Need to use complete sentences in present tense.